data/templates/frr/ospfd.frr.j2

```diff
@@ -170,7 +170,7 @@ router ospf {{ 'vrf ' ~ vrf if vrf is vyos_defined }}
 {% if parameters.router_id is vyos_defined %}
  ospf router-id {{ parameters.router_id }}
 {% endif %}
-{% if passive_interface.default is vyos_defined %}
```

Comment:
Please use instead: `{% if passive_interface is vyos_defined('default') %}`

Reply:
done